src/status_im/contexts/communities/overview/style.cljs

```diff
@@ -21,6 +21,9 @@
    :align-items    :center
    :margin-top     20})
 
+(def tmp-fetching
+  {:align-items :center :top "50%"})
```

Comment:
Can we use
```
:align-items :center
:justify-content :center
```

Reply:
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#dont-use-percents-to-define-widthheight (probably same guidelines apply for :top)